Stigbd/issue4-Add-support-for-language-negotiation-base-on-accept-language-header (#5)
Informasjonsforvaltning/content-negotiation: README.md

[![PyPI](https://img.shields.io/pypi/v/content-negotiation.svg)](https://pypi.org/project/content-negotiation/)
[![Read the Docs](https://readthedocs.org/projects/content-negotiation/badge/)](https://content-negotiation.readthedocs.io/)

A small Python library supporting content-negotiation.

> It can be used to deciding content type based on a list of media ranges, as well as deciding content-language based on the accpe-language header.

Reviewer: ```suggestion
It is used to decide content type based on a list of media ranges, as well as deciding content-language based on the accpe-language header.
```

    # Handle error, by returning e.g. 406 Not Acceptable
```

#### Content language

```Python
from content_negotiation import decide_content_language, NoAgreeableContentLanguageError

accept_language_headers = ["application/json", "text/html", "text/plain, text/*;q=0.8"]

> supported_languages = ["text/turtle", "application/json"]

Reviewer: Var ikke klar over at turtle var et språk :joy:

Author: Fiksa i 092d21b
